systemjs/react-hello-world/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>SystemJS AMD Modules from CDN Example</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="systemjs-importmap">
+    {
+      "imports": {
+        "react": "https://cdn.jsdelivr.net/npm/react/umd/react.production.min.js",
+        "react-dom": "https://cdn.jsdelivr.net/npm/react-dom/umd/react-dom.production.min.js"
+      }
+    }
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/amd.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/use-default.min.js"></script>
```

Comment:
Do we really need the AMD and use default plugins for this example to work?

Reply:
Yes -- see https://cdn.jsdelivr.net/npm/react/umd/react.development.js. We're loading react from CDN, and React does not have a System.register version available on npm.

Reply:
Have you tested this? Surely the global extension works fine?

Reply:
ah yeah I didn't think to use the global extension. I'll try that out now.

Reply:
Yep the global loading works fine -- updated to remove the extras